pkg/graph/resolvers/trb_request_feedback.go

```diff
@@ -39,7 +39,9 @@ func CreateTRBRequestFeedback(
 		}
 
 		for _, recipientInfo := range recipientInfos {
-			recipientEmails = append(recipientEmails, recipientInfo.Email)
+			if recipientInfo != nil {
```

Comment:
What was the point of the nil check here?

Reply:
@mynar7 fetchUserInfos returns a slice of pointers, and those pointers could be nil. The issue came up when I was testing the endpoint to get emails to generate, and some of my EUAs that I used in the graphQL playground produced nil values for recipientInfo.